airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -61,9 +61,15 @@ definitions:
         error_message: >-
           HubSpot server error (5xx). Retrying with exponential backoff...
       - type: HttpResponseFilter
-        action: FAIL
+        action: RETRY
```

Comment:
should we add `RETRY` action for dynamic streams error handler [here](https://github.com/airbytehq/airbyte/blob/b301837508904579c7f9a1aa7c10009c06f2a4b7/airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml#L2081)?